.djlintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "blank_line_after_tag": "load,extends",
+  "blank_line_before_tag": "load,extends,block, macro",
+  "close_void_tags": true,
+  "ignore": "J018,H006,H021,H023,H029,T002,T003,T028",
+  "include": "H017",
```

Comment:
It's probably out of the scope of this PR, but maybe something we discuss at dev catchup to decide on the rules we want to enable/disable.

Judging on our experience with prettier it's easier and better to just follow the defaults and adjust our own internal rules to follow. So if the default is not to close void tags, maybe we can accept it and change our HTML guidelines, so that we don't have to make exceptions for something that is accepted as common standard.

Reply:
Okay, shall I bring up this topic to the dev catchup meeting again, but this time go through each rule that was added/ignored?
I'm just afraid that we might end up never having it cause the discussion will keep going infinitely.
What about we merge it the way it is now, and if later someone is uncomfortable about a certain rule, they can bring it up to discussion themselves?

Reply:
I don't mean to block this on having the dev catchup discussion.

Let's merge this, start using it. But before we make it as a cross team tool with some shared config would be good to discuss is we need any exceptions from defaults.